apps/new-dealer/views/authority-holders-nationality.html

```diff
@@ -1,14 +1,16 @@
 {{<partials-page}}
   {{$page-content}}
     {{#select}}{{key}}{{/select}}
-    {{#checkbox}}{{key}}-multi{{/checkbox}}
+    <div data-module="govuk-checkboxes">
+      {{#checkbox}}{{key}}-multi{{/checkbox}}
```

Comment:
This is interesting can you explain this?

Reply:
this needs to be data-module?

Reply:
I added data-module according to the govuk design because the multiple nationalities would not toggle. This could also have been achieved by adding a child config in fields/index.js  directed at partials/ multi-nationality.html, amending the partials/multi-nationality.html to include the select mixins and removing the select mixins from authority-holders-nationality.html.